contracts/BinaryOptionMarketData.sol

```diff
@@ -215,13 +221,15 @@ contract BinaryOptionMarketData {
 
         (IBinaryOption long, IBinaryOption short) = market.options();
         (uint biddingEndDate, uint maturityDate, uint expiryDate) = market.times();
+        (uint price, uint updatedAt) = market.oraclePriceAndTimestamp();
```

Comment:
This is causing a stack too deep error. Moving it to the `getMarketData` function would solve the issue.